### ISSUE TITLE
Cursor width when indent using tabs

### ISSUE THREAD

**Issue by duranivan** [VSCodeVim/Vim#3856]
**Describe the bug**
Basically, when the file is using indentation by tabs, the cursor width in normal mode changes according to the tab size (e.g. 2 or 4 spaces). I know it is not a big issue but somehow I find it a bit annoying. Here is an example of how it looks:

![Screen Shot 2019-06-27 at 11 04 50 PM](https://user-images.githubusercontent.com/20104703/60289528-10bfdc80-9930-11e9-8860-b9d7f2e84812.png)

In the picture, there are two cursors, one with width equal to 2 spaces due to the tab indentation, and one with 1 space width (the one in between `if` and `len`).

If I was using tab indentation with 4 spaces, then the difference between the two cursors would be more dramatic as you can imagine.

**Expected behavior** 

So basically, the cursor width should always be the same (one space width), regardless whether it is on a tab indentation or not. This is the case in other editors vim extensions (e.g. ideavim or neovintageous).

Thanks in advance!

**Comment by J-Fields** [comment]
Disabling the extension and setting `editor.cursorStyle` to "block" has this behavior too. I believe it's intentional (and useful), so I'm not sure this should be changed.

**Comment by duranivan** [comment]
Oh, really? I have used VSCode with the Vim extension since the beginning so I didn't know that this behavior was from VSCode itself. I can see why one would find it as useful, but once again, in other editors, this is not the case. I personally I find it not appealing at all nor useful in my case.

In any case, I'm closing this issue since it is not from the Vim extension.

Thanks!

**Comment by xh3b4sd** [comment]
I just moved to vsCode and find the existing behaviour quite irritating. I cannot find any configuration to keep the cursor block width constant. How is this supposed to be?

**Comment by J-Fields** [comment]
@xh3b4sd This is just how VSCode renders a block cursor on a wide character. Honestly don't really desire why it wouldn't be desirable, but you can file an issue in the VSCode repo or switch to using a different cursor style (line for instance).
